proto/api_test.go

```diff
@@ -150,10 +150,11 @@ func TestCombinable(t *testing.T) {
 
 func TestSetGoErrorCopy(t *testing.T) {
 	rh := ResponseHeader{}
-	err := &Error{Message: "test123"}
+	err := &RangeKeyMismatchError{RequestStartKey: Key("test123")}
+	errStr := err.Error()
 	rh.SetGoError(err)
-	err.Message = "321tset"
-	if rh.Error.Message != "test123" {
+	err.RequestStartKey = Key("321tset")
```

Comment:
all that matters for this test is that we use an error which has something in it that we can change. I'm using a `RangeKeyMismatchError` because it's convenient.